api/routers/cameras.py

```diff
@@ -340,3 +342,47 @@ async def remove_roi_contour(camera_id: str, reboot_processor: Optional[bool] =
     os.remove(roi_file_path)
     success = restart_processor() if reboot_processor else True
     return handle_response(None, success, status.HTTP_204_NO_CONTENT)
+
+
+@cameras_router.get("/{camera_id}/in_out_boundaries")
+async def get_in_out_boundaries(camera_id: str):
+    """
+        Get the In/Out Boundaries
+    """
+    in_out_file_path = InOutMetric.get_in_out_file_path(camera_id, settings.config)
+    in_out_boundaries = InOutMetric.get_in_out_boundaries(in_out_file_path)
+    if in_out_boundaries is None:
```

Comment:
I think that before checking for the boundaries we should check for camera existence in order to return a clear 404 message.

Reply:
Huh, I thought we were doing that by default on a callback, I'll update the endpoints to check for camera existance